docs/split-inputs.md

```diff
@@ -136,8 +136,22 @@ and split the fuzz input into several parts, the corpus elements will no longer
 be valid image files, and you won't be able to simply add image files to your
 corpus.
 
-[FuzzedDataProvider] is a class whose constructor accepts `const uint8_t*,
-size_t` arguments. Below is a quick overview of the available methods.
+### Main concepts
+
+* [FuzzedDataProvider] is a class whose constructor accepts `const uint8_t*,
+  size_t` arguments. Usually, you would call it in the beginning of your
+  `LLVMFuzzerTestOneInput` and pass `data, size` provided by the fuzzing engine.
```

Comment:
nit: "the `data, size` parameter" or similar

Reply:
Done